src/functionalTest/java/uk/gov/hmcts/reform/ccd/document/am/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.ccd.document.am.befta;
```

Comment:
Package naming needs adjustment.

Reply:
Build is failing. Not able to update this comment at the moment.
Will update this change later. 
Execution failed for task ':checkOutcomes'.
> No such property: toUri for class: sun.nio.fs.UnixPath